sktime/utils/_testing/estimator_checks.py

```diff
@@ -316,8 +324,10 @@ def check_fit_updates_state(Estimator):
         assert not getattr(
             estimator, attr
         ), f"Estimator: {estimator} does not initiate attribute: {attr} to False"
-
-    fit_args = _make_args(estimator, "fit")
+    n_columns = _get_n_columns(
+        estimator.get_tag(tag_name="scitype:y", raise_error=False)
```

Comment:
this should not override `raise_error`, so it's caught in case the tag name changes in the future

Reply:
but we need it, as not all estimators have this tag but we are iterating over them

Reply:
ideally, `raise_error=False` will still raise an error if this tag does not exist at all in `sktime`. Currently if `raise_error=True` it just compares with the given estimator tag.

Reply:
but that's another topic outside this PR?